src/hooks/useLoadGlobalLateData.ts

```diff
@@ -50,7 +46,7 @@ export default function useLoadGlobalLateData() {
     // tx method names
     const p7 = dispatch(transactionSignaturesLoadState());
 
-    promises.push(p1, p2, p3, p4, p5, p6, p7);
+    promises.push(p2, p3, p4, p5, p6, p7);
```

Comment:
can rename these but meh, we should slowly try to remove all of these

Reply:
nitpick but can you rename these to p1, p2, p3, p4, p5, p6